src/utils.ts

```diff
@@ -25,6 +26,34 @@ export function pathExists(p: string): boolean {
   return true;
 }
 
+export async function initConfig(rootPath: string) {
+  const terminal = findOrCreateTerminal();
+  if (!pathExists(rootPath + '/tmp')) {
+    terminal.sendText('mkdir tmp');
+  }
+  terminal.sendText('afconfig view --yaml > tmp/merged.yaml');
+  // This blocks during the initial activation of the extension,
+  // to allow time for the terminal command to complete and hydrate values
+  // Post activation of the extension, this sleep fires but doesn't block
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+}
+
```

Comment:
This feels like the most fragile piece.   During extension initialization, the sleep blocks so that the very first config initialization at least has a file at the path.  This is important since the filewatcher isn't initialized yet, so we get no event that the file changed once the terminal command finishes.

Any edits to config or the file after extension initialization don't pay the sleep, it fires the sleep and doesnt block.